src/form/form.tsx

```diff
@@ -88,6 +88,7 @@ interface Props<MODEL> {
     validationSchema?: any;
     validationOptions?: ValidationOptions;
     resolveViolationMessage?: ViolationMessageResolver;
+    createButtonFactories?: Array<(buttonProps: {}) => React.ReactNode>;
```

Comment:
I know it was mine, but doesn't sth like `additionalButtonRenderers` fit a bit more the existing wording?